action.yml

```diff
@@ -56,5 +56,5 @@ outputs:
     description: The API url of the deployment statuses.
 
 runs:
-  using: 'node12'
+  using: 'node16'
```

Comment:
This needs to be Node 20 now to avoid the deprecation warnings: https://github.blog/changelog/2023-09-22-github-actions-transitioning-from-node-16-to-node-20/

Reply:
It's been 1.5 years, I sadly don't think this PR will get merged 😢